openedx/adg/lms/webinars/models.py

```diff
@@ -58,13 +60,17 @@ def upcoming_webinars(self):
         """
         Filter queryset to display upcoming webinars
         """
-        return self.filter(is_cancelled=False, is_published=True, end_time__gte=now())
+        return self.annotate(
+            webinar_end_time=ExpressionWrapper(F('start_time') + F('duration'), output_field=DateTimeField())
+        ).filter(is_cancelled=False, is_published=True, webinar_end_time__gte=now())
 
     def delivered_webinars(self):
         """
         Filter queryset to display delivered webinars
         """
-        return self.filter(is_cancelled=False, end_time__lt=now())
+        return self.annotate(
+            webinar_end_time=ExpressionWrapper(F('start_time') + F('duration'), output_field=DateTimeField())
```

Comment:
`self.annotate(webinar_end_time=ExpressionWrapper(F('start_time') + F('duration'), output_field=DateTimeField()))` is being repeated twice. Can we somehow avoid this duplication, by separating out the ExpressionWrapper and using it in both places?